jsonpointer.py

```diff
@@ -84,12 +94,14 @@ def set_pointer(doc, pointer, value, inplace=True):
     True
     """
 
-    pointer = JsonPointer(pointer)
-    return pointer.set(doc, value, inplace)
+    _pointer = JsonPointer(pointer)
+    return _pointer.set(doc, value, inplace)
```

Comment:
Some type engines (e.g. pylance) get confused when you reuse names in place like this